README.md

```diff
@@ -21,17 +21,17 @@ These instructions will get you a copy of the role for your ansible playbook. On
 
 ### Prerequisities
 
-Ansible 2.2.1.0 version installed.
+Ansible 2.4.0.0 version installed.
 Inventory destination should be a Debian environment.
 
-For testing purposes, [Molecule](https://molecule.readthedocs.io/) with [Vagrant](https://www.vagrantup.com/) as driver (with [landrush](https://github.com/vagrant-landrush/landrush) plugin) and [VirtualBox](https://www.virtualbox.org/) as provider.
+For testing purposes, [Molecule](https://molecule.readthedocs.io/) with [Vagrant](https://www.vagrantup.com/) as driver (with [hostsupdater](https://github.com/cogitatio/vagrant-hostsupdater) plugin) and [VirtualBox](https://www.virtualbox.org/) as provider.
 
 ### Installing
 
 Create or add to your roles dependency file (e.g requirements.yml):
 
 ```
-- src: idealista.newrelic-role
+- src: idealista.newrelic-servers-agent-role
   version: 1.0.0
   name: newrelic
```

Comment:
The name should be `newrelic-servers-agent`, as in L50 ;)

Reply:
done